src/modules/endpoint/mappers/categories.js

```diff
@@ -0,0 +1,36 @@
+import CategoryModel from '../models/CategoryModel'
+import CategoriesMapModel from '../models/CategoriesMapModel'
+
+function categoriesMapper (json, params) {
```

Comment:
We try to avoid function declarations. Use a lambda instead. I think I forgot to add a linter rule for this :see_no_evil:

Reply:
I thought we try to avoid lambdas :D will be changed